### PR TITLE
CB-18041 : Removed vmagent related code

### PR DESCRIPTION
**Pull request by szabolcs-horvath**
Before we introduced prometheus agent we used vmagent to report metrics to Thanos. Since monitoring is not in production we can safely remove vmagent related code from cloudbreak such as salt states, salt related config generation in java.

### CODE REVIEW

**Review by foldik — APPROVED** [review]
LGTM

**Review by oleewere — APPROVED** [review]

File: common/src/main/java/com/sequenceiq/cloudbreak/telemetry/TelemetryUpgradeConfiguration.java [hunk 1]
```diff
@@ -15,8 +15,6 @@ public class TelemetryUpgradeConfiguration {
 
     private TelemetryComponentUpgradeConfiguration meteringAgent;
 
-    private TelemetryComponentUpgradeConfiguration cdpVmAgent;
```
Comment (oleewere) [inline common/src/main/java/com/sequenceiq/cloudbreak/telemetry/TelemetryUpgradeConfiguration.java:18]: sorry one thing that i have missed, this config is referenced here as well: https://github.com/hortonworks/cloudbreak/blob/master/common/src/main/resources/common-config.yml#L88

**Review by oleewere — CHANGES_REQUESTED** [review]

**Review by oleewere — APPROVED** [review]

**Review by lacikaaa — CHANGES_REQUESTED** [review]
could you please elaborate in the commit message a bit more? the jira task has a good description, you can copy that one

**Comment by lacikaaa** [comment]
Editing github pr description is not the same as commit message. Please override your commit message with the `git --amend` command and then force push your branch

**Review by lacikaaa — APPROVED** [review]

**Pull request merged**